modules/repository/repo.go

```diff
@@ -182,9 +182,10 @@ func CleanUpMigrateInfo(repo *models.Repository) (*models.Repository, error) {
 // SyncReleasesWithTags synchronizes release table with repository tags
 func SyncReleasesWithTags(repo *models.Repository, gitRepo *git.Repository) error {
 	existingRelTags := make(map[string]struct{})
-	opts := models.FindReleasesOptions{IncludeDrafts: true, IncludeTags: true}
+	opts := models.FindReleasesOptions{IncludeDrafts: true, IncludeTags: true, ListOptions: models.ListOptions{PageSize: 50}}
```

Comment:
I agree that 50 seems more reasonable than 100 here.

Reply:
Actually I didn't intend to change that value... Should I leave 50 or make it 100 to make sure it won't break anything?

Reply:
Nah 50 is more reasonable I think